website/src/theme/Footer/index.js

```diff
@@ -88,17 +88,18 @@ function Footer() {
             ))}
           </div>
         )}
-        {(logo || copyright) && (
```

Comment:
I opted to remove this and put the conditional logic around the copyright instead as logo already has conditional logic.